api_methods.py

```diff
@@ -0,0 +1,132 @@
+"""Python auto.ria.com API.
+
+Python implementation of API intended for calulating
+average used car prices that are sold on http://auto.ria.com
+"""
+
+
+import requests
+import json
+from fnmatch import fnmatch
+
+
+def jsonify_response(url):
+    """Send get request and return data in JSON."""
+    response = requests.get(url)
+    if response.status_code == 200:
+        return json.loads(response.text)
+
+
+def get_categories():
+    """Get available vehicle types from auto.ria.com."""
+    url = 'http://api.auto.ria.com/categories'
+    return jsonify_response(url)
+
+
+def get_bodystyles(category):
+    """Get available bodystyles from auto.ria.com."""
```

Comment:
The docstring should also describe parameters and return value. If we use google style (http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html), it will be like this:

```
def get_bodystyles(category):
    """Get available bodystyles from auto.ria.com.

    Args:
        category - string, category name, from the list returned by
            ``get_categories`` method.

    Returns:
        List of strings - body styles like `X` or `Y` (having examples of real values instead of X and Y)
    """
```

Even better, also use type hints (https://www.python.org/dev/peps/pep-0484/, https://docs.python.org/3/library/typing.html) to make it clear the in / out parameter types:

```
def get_bodystyles(category: str) -> List[str]:
    """Get available bodystyles from auto.ria.com.

    Args:
        category - category name, from the list returned by ``get_categories`` method.

    Returns:
        List of body styles like `X` or `Y`.
    """
```